hash/spectrum_cass.xml

```diff
@@ -171755,7 +172272,7 @@ Tape 3 - Vendetta (System 3 Software)
 	<software name="testmatc">
 		<description>Test Match &amp; One Day Cricket</description>
 		<year>1983</year>
-		<publisher>CRL Group</publisher>
+		<publisher>CRL Group PLC</publisher>
```

Comment:
Why did you add the “PLC” company structure suffix for CRL group (Proprietary Limited Company), when you’re removing company structure suffixes from other publisher names (e.g. removing the “Ltd” suffix for Personal Software Services)?

Reply:
You're right. Removed the PLC suffix.